File: API/TaskManegement.Model/Model/TaskModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManagement.Model.Model
{
    public class TaskModel
    {
        public long TaskId { get; set; }
        public string? TaskName { get; set; }
        public string? Description { get; set; }
        public DateTime Deadline { get; set; }
        public long UserId { get; set; }
        public string? Priority { get; set; }   
    }
}

File: TaskManagement_Admin/src/app/core/model/task-model.ts
export interface TaskModel{
    TaskId : number,
    TaskName : string,
    Description : string,
    Deadline : any,
    UserId : number,
    Priority : string
}

export interface TimeLogModel {
    TimeLogId : number,
    TimeLogText : string,
    TaskId : number,
    UserId : number,
    StartTime : any,
    EndTime : any,
    TaskName : string,
    ParentId : number,
}